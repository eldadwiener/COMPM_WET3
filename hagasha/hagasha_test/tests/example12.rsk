<header>
<unimplemented>
<implemented> main,12 noRet,1
</header>
PRNTC 105
PRNTC 110
PRNTC 32
PRNTC 110
PRNTC 111
PRNTC 82
PRNTC 101
PRNTC 116
PRNTC 10
COPYI I2 I1
RETRN
PRNTC 10
PRNTC 84
PRNTC 101
PRNTC 115
PRNTC 116
PRNTC 58
PRNTC 32
PRNTC 105
PRNTC 110
PRNTC 116
PRNTC 32
PRNTC 102
PRNTC 117
PRNTC 110
PRNTC 99
PRNTC 32
PRNTC 119
PRNTC 105
PRNTC 116
PRNTC 104
PRNTC 111
PRNTC 117
PRNTC 116
PRNTC 32
PRNTC 114
PRNTC 101
PRNTC 116
PRNTC 117
PRNTC 114
PRNTC 110
PRNTC 32
PRNTC 115
PRNTC 116
PRNTC 97
PRNTC 116
PRNTC 101
PRNTC 109
PRNTC 101
PRNTC 110
PRNTC 116
PRNTC 10
STORI I0 I2 0
STORI I1 I2 4
STORI I2 I2 8
CITOF F0 I2
ADD2I I2 I2 16
COPYI I1 I2
JLINK 1
SUBTI I2 I2 16
LOADI I0 I2 0
LOADI I1 I2 4
LOADI I2 I2 8
LOADI I3 I2 12
COPYI I2 I1
RETRN
HALT
