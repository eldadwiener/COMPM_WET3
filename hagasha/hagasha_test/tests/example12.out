
Test: int func without return statement
in noRet
Reached Halt.
